addons/project/views/project_sharing_templates.xml

```diff
@@ -12,7 +12,7 @@
 
     <template id="project_sharing" name="Project Sharing View">
         <!--    We need to forward the request lang to ensure that the lang set on the portal match the lang delivered -->
-        <iframe class="flex-grow-1" frameborder="0" t-attf-src="/{{ request.context['lang'] }}/my/projects/{{ str(project_id) }}/project_sharing{{ '?task_id=' + task_id if task_id else '' }}"/>
+        <iframe class="flex-grow-1" frameborder="0" t-attf-src="/my/projects/{{ str(project_id) }}/project_sharing{{ '?task_id=' + task_id if task_id else '' }}"/>
```

Comment:
Comment was there for this, remove it too :)
If you are sure it's not needed as it was explicitly added to fix something: https://github.com/odoo/odoo/pull/91485

Reply:
I removed it because it was buggy; `request.context['lang']` is a code, not an url_code (+nothing guarantees that this lang of the user actually exists for the website). Have to test about portal lang redirection though.